about_page.html

```diff
@@ -0,0 +1,119 @@
+            <div>
+                <div class="about-info">
+                    <h1>INNOVATIONX COMMONS</br>AFRICA SUMMIT 2023</h1>
+                    <p class="welcome-abt">
+                        Welcome to InnovationX Summit Africa, the premier conference for innovation in Africa. This annual event brings together entrepreneurs, investors, academics, and policymakers to share insights and ideas about the latest trends and best practices in innovation. Our mission is to accelerate innovation in Africa and drive economic growth by connecting innovators with the resources and support they need to succeed.
+                    </p>
+                    <div>
+                        <p class="welcome-blck">
+                            Please contact us per email on any further questions about the InnovationX Africa Summit 2023! </br>
+                            <span>
+                                miyoyo@gmail.com
+                            </span>
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="first-description">
+            <div class="second-description">
+            <h3>InnovationX Commons Africa Summit Logo</h3>
+            <p>
+                The InnovationX Commons Africa Summit logo was designed by a team of graphic designers who drew inspiration from the vibrant colors and patterns of African textiles and art. The logo symbolizes the diverse and interconnected community of innovators that the summit brings together to collaborate and create positive change in Africa.
+            </p>
+            <div class="logo-abt">
+                <img src="./images/logos/CSi.png" />
+            </div>
+        </div>
+    </section>
+    <hr class="brbr" />
+    <section class="past">
+        <h3>See Past InnovationX Commons Africa Summit Fora</h3>
+        <p>
+            Take a look at the last two InnovationX Commons Africa Summit fora that happened in Bueno Aires and in Warsaw.
+        </p>
+        <div class="previous-summits">
+            <img src="./images/summit/1.png" />
+            <img src="./images/summit/2.png" />
+        </div>
+    </section>
+    <section class="sponsors">
+        <h2>Partner</h2>
+        <div class="sponsor">
+            <img src="./images/partners/youtube.png" alt="YouTube" width="100px" />
+            <img src="./images/partners/facebook.png" alt="Meta" width="100px" />
+            <img src="./images/partners/windows.png" alt="Microsoft" width="100px" />
+            <img src="./images/partners/ford.png" alt="Ford Motors" width="100px" />
+            <img src="./images/partners/grand.png" alt="Ford Motors" width="100px" />
+        </div>
+    </section>
+    <section class="footer">
+        <div>
+            <img src="./images/logos/CSi.png" width="30px" />
+        </div>
+        <div>&copy 2023 - InnovationX Summit</div>
+    </section>
```

Comment:
- Hey, in here, this is the footer of your page which should be a `<footer>` tag and not a `<section>`.  As the footer is meant to be a standalone element in your webpage, you should move it out of this section. kindly update this to improve your semantic usage and to improve your page layout.